ember-core/shared/src/test/scala/org/http4s/ember/core/EncoderSuite.scala

```diff
@@ -125,6 +126,32 @@ class EncoderSuite extends Http4sSuite {
     Helpers.encodeResponseRig(resp).assertEquals(expected)
   }
 
+  test("respToBytes should encode a no body response correctly with no header") {
+    val resp = Response[IO](Status.Ok)
+
+    val expected =
+      """HTTP/1.1 200 OK
+      |Content-Length: 0
+      |
+      |""".stripMargin
+
+    Helpers.encodeResponseRig(resp).assertEquals(expected)
+  }
+
+  test("respToBytes should encode a no body response correctly with stream") {
+    val resp = Response[IO](Status.Ok, body = Stream.chunk(Chunk.empty))
+
+    val expected =
+      """HTTP/1.1 200 OK
+      |Transfer-Encoding: chunked
+      |
+      |0
+      |
+      |""".stripMargin
+
+    Helpers.encodeResponseRig(resp).assertEquals(expected)
+  }
```

Comment:
So is this intended behavior? I guess it's the only implementation that makes sense.

Reply:
Yeah, I don't think we can do better than this.  And even though it's arguably overspecified, I like the test to alert us to behavior changes.